src/frontend/components/SøknadsSteg/OmBarnet/OmBarnet.test.tsx

```diff
@@ -45,6 +45,11 @@ const jens = {
     [barnDataKeySpørsmål.boddMindreEnn12MndINorge]: { id: '14', svar: ESvar.NEI },
     [barnDataKeySpørsmål.barnetrygdFraAnnetEøsland]: { id: '15', svar: ESvar.JA },
     [barnDataKeySpørsmål.barnetrygdFraEøslandHvilketLand]: { id: '16', svar: 'AUS' },
+    [barnDataKeySpørsmål.mottarEllerMottokEøsBarnetrygd]: {
+        id: '166',
+        svar: ESvar.NEI,
+    },
+    eøsBarnetrygdsperioder: [],
```

Comment:
Hmm, finnes dette feltet nå? ref de andre kommentarene til renaming for heter ikke denne nå registrerteEøsBarnetrygdsperioder ? Men hvis du renamer de andre til det her så er det jo good 😊